roles/ood_jupyter/templates/custom-jupyter-form.yml

```diff
@@ -37,8 +37,12 @@ attributes:
     widget: text_area
     label: Environment Setup
     value: |
-      # Load required modules
-      module load Anaconda3
+      # By default, it already load latest version of Anaconda3
```

Comment:
Change line text to:
"The latest version of Anaconda3 with jupyter is loaded by default."